.github/workflows/mingw-build.yml

```diff
@@ -38,10 +38,13 @@ jobs:
             -DENABLE_TOOLS=ON \
             -DENABLE_DATA_TOOLS=OFF \
             -DENABLE_SERVICES=OFF \
-            -DENABLE_PYTHON_BINDINGS=OFF \
+            -DENABLE_PYTHON_BINDINGS=ON \
             -DENABLE_CCACHE=OFF \
             -DENABLE_TESTS=OFF \
             -DENABLE_BENCHMARKS=OFF \
             -DLOGGING_LEVEL=DEBUG \
-            -DBoost_PROGRAM_OPTIONS_LIBRARY=/usr/x86_64-w64-mingw32/sys-root/mingw/lib/libboost_program_options-mt-x64.dll.a && \
+            -DBoost_PROGRAM_OPTIONS_LIBRARY=/usr/x86_64-w64-mingw32/sys-root/mingw/lib/libboost_program_options-mt-x64.dll.a \ 
+            -DPYTHON_EXECUTABLE=/usr/x86_64-w64-mingw32/bin/python3 \
+            -DPYTHON_LIBRARIES=/usr/x86_64-w64-mingw32/sys-root/mingw/lib/libpython3.9.dll.a \
+            -DPYTHON_INCLUDE_DIRS=/usr/x86_64-w64-mingw32/sys-root/mingw/include/python3.9 && \
```

Comment:
@nilsnolde no, but close, 😄 , it's not the trailing slash, but the syntactic incorrect ` && ` before the `\`

Reply:
huh? haha seriously I don't see the problem @mloskot .. There's another command following the `&&`, really am I blind? not only that I don't see it, but my local setup is the exact same (in .sh form) and no syntax errors..

Reply:
I give up.. in this commit it "worked": https://github.com/valhalla/valhalla/blob/3b3ef50c3cef5fb12545a21cc28ef43509ff1d8d/.github/workflows/mingw-build.yml

And that's no different what I've been doing with the very last commit: https://github.com/valhalla/valhalla/blob/5b43e6c1e3d3a9cf2fee87542db8aa3c4d2a4fef/.github/workflows/mingw-build.yml

First goes through, last one doesn't! wtf..

Reply:
Sorry, I just looked at the diff part, drawing conclusions based on  incomplete picture and a minute before calling it a day. I should have shut up :)

No idea what's going wrong there but I wouldn't be surprised if it was a ln issue with parsing YAML with long complex shell command, perhaps hitting some sort of bug or limit. I've seen YAML issues on Azure Pipelines.

Reply:
Well, no kidding, whitespace was hidden there behind the last `\`!!! AAAARGGH! Cost me a good 2 hours of my life.. Thanks @timmccauley for spotting that!

Reply:
Ha, good one!